packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -59,6 +64,11 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     /// @notice An append-only array of all claims made during the dispute game.
     ClaimData[] public claimData;
 
+    /// @notice The starting and disputed output proposal for the game. Includes information about
+    ///         the output indexes in the `L2OutputOracle` and the output roots at the time of
+    ///         game creation.
+    OutputProposal[2] public proposals;
```

Comment:
nit: Much clearer to use separate fields wit descriptive names (`agreedProposal`, `disputedProposal`) rather than a fixed array.

Reply:
its the same amount of storage, i suppose just wanting to do it in a single call to get both? More descriptive does make sense

Reply:
Yeah, and you still need to specify the array index to access it anyways.